### PR TITLE
Upgrade to Crowdin V2 APIs

### PR DESCRIPTION
**Pull request by riknoll**
Upgrading our Crowdin support to use the V2 APIs.

I've done a little bit of testing and things seems to be running as they should. I was able to download strings from arcade and upload a few (unchanged) files to pxt-ev3. In any case, this is safe to merge since we don't have any updated API tokens in place. I plan to test this with pxt-ev3 first since that target has fewer eyes on it.

This PR also removes support for crowdin branches. AFAIK we don't actually use them anywhere (@pelikhan correct me if I'm wrong) and they are not compatible with the directory based structure in the V2 api, so I don't expect this to be an issue. I still need to pull out a lot of references to crowdin branches in the code but I'll do that in a separate PR.

Also, I have temporarily disabled the `pxt crowdin stats` command but I've got a plan to bring it back.

### CODE REVIEW

**Comment by pelikhan** [comment]
Yup not used

**Comment by pelikhan** [comment]
My scripts use their cli

File: cli/crowdinApi.ts [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+    const storageResponse = await uploadStorageApi.addStorage(fileName, fileContent);
+
+    await sourceFilesApi.updateOrRestoreFile(projectId, fileId, {
+        storageId: storageResponse.data.id,
+    });
+}
+
+function getClient() {
+    if (!client) {
+        client = new crowdin(crowdinCredentials());
```
Comment (jwunderl) [inline cli/crowdinApi.ts:345]: might want to pass opts with a retryConfig as there have historically been a decent number of spurious errors (pass that in pxt-translations along with ignoring 304's and it was very necessary at the time at least)

**Review by jwunderl — APPROVED** [review]

**Comment by ganicke** [comment]
Gonna tag this one to close #9539

**Pull request merged**